python/graphstorm/gconstruct/remap_result.py

```diff
@@ -49,6 +49,87 @@
 # id_maps to each worker process.
 id_maps = {}
 
+def worker_remap_node_emb(emb_file_path, nid_path, nid_range_info, ntype,
+    output_fname_prefix, chunk_size, preserve_input):
+    """ Do one node prediction remapping task
+
+        GraphStorm has two ways to store node embeddings:
+        1. Store the entire node embeddings of nodes of ntype T.
+           In this case, node embeddings has already been shuffled
+           into the order before doing graph partition.
+           The order is aligned with the order in the
+           original-to-graphstorm ID mapping.
+           Link prediction inference, edge prediction inference and
+           embedding inference usually generate node embeddings in this way.
+        2. Store the node embeddings of a subset of nodes of ntype T.
```

Comment:
should we store node embeddings in the same way?

Reply:
The first one is more memory efficient.
The second one works in cases when only subset of node embeddings are saved.

Reply:
Unify the way how we save node embeddings and predictions.